app/src/utils/ethers.ts

```diff
@@ -27,5 +27,5 @@ export { BigNumber } from '@ethersproject/bignumber';
 export { hexDataSlice } from '@ethersproject/bytes';
 export { Contract } from '@ethersproject/contracts';
 export { JsonRpcProvider, JsonRpcSigner, Web3Provider } from '@ethersproject/providers';
-export { commify, formatUnits, parseUnits } from '@ethersproject/units';
+export { commify, formatEther, formatUnits, parseUnits } from '@ethersproject/units';
```

Comment:
Looks like this export was added but I don't see it used

Reply:
It was used when were showing the ETH total, but still not clear if we are going to show it. I'll remove it for now.